workorders.lic

```diff
@@ -260,15 +260,22 @@ class WorkOrders
   def shape_items(info, item, quantity)
     ensure_copper_on_hand(5000, @hometown)
     recipe, items_per_stock, spare_stock, scrap = find_recipe(info, item, quantity)
+    stock_name = 'stock-name'
```

Comment:
What do you think about `stock_name = info['stock-name']` instead?

Reply:
When I change these two to that, they end up being blank. It'll do a get my (extra space here) lumber instead of get my maple lumber.

Reply:
That usually indicates a typo in the thing you're using for the lookup. Maybe `-` (dash) vs `_` (underscore)?